test/fixtures/app.js

```diff
@@ -6,6 +6,10 @@
 var loopback = require('loopback');
 var app = loopback();
 
+app.dataSource('db', { connector: 'memory' });
+var User = loopback.createModel('User');
+app.model(User, { dataSource: 'db' });
+
```

Comment:
This is fine. Can you please add a comment explaining that this is needed in order to have the script long enough to trigger output truncation?